.travis.yml

```diff
@@ -16,3 +16,4 @@ install:
 
 script:
     - go test -v ./...
+    - go test -run=^$ -bench BenchmarkReference -benchmem
```

Comment:
Really minor and optional, but I'd consider doing including `=` in `-bench=BenchmarkReference` for consistency with `-run=^$`. Up to you.